### PR TITLE
Add clear button funtionality and Fix some style changes

### PR DESCRIPTION
**Pull request by KmBarnett**
Done with this part changed some html to make it work buttons into input types and css for active became disabled and active background changed.

### CODE REVIEW

**Comment by KmBarnett** [comment]
Master is the same as it was when we split yesterday

File: main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+  function clearFields() {
+    for (var i = 0; i < input.length; i++) {
+      input[i].value = " "
+    }
+    clearButton.setAttribute('disabled', true)
+  }
+
+  input[0].addEventListener('input', enableButton)
```
Comment (Karlfunhouse) [inline main.js]: Kyle is going to change the variable input to something else that isn't already a javascript event title.

**Comment by KmBarnett** [comment]
I Realized now that i also did iteration 1.3

File: styles.css [hunk 2]
```diff
@@ -138,7 +142,7 @@ h1 {
 }
 
 .game-button {
-  background-color: #d0d2d3;
```
Comment (Karlfunhouse) [inline styles.css:141]: Changed background color to be darker and added a disabled class that will be the lighter color naturally until the appropriate input has content and then the disabled class will be removed

**Review by Karlfunhouse — CHANGES_REQUESTED** [review]

File: main.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  function clearFields() {
+    for (var i = 0; i < challengerInputField.length; i++) {
+      challengerInputField[i].value = " "
+    }
+    clearButton.setAttribute('disabled', true)
+  }
+
+  challengerInputField[0].addEventListener('input', enableButton)
+  challengerInputField[1].addEventListener('input', enableButton)
```
Comment (KmBarnett) [inline main.js:17]: changed variable input to  challengerInputField

File: main.js (hunk 1 above)
Comment (KmBarnett) [reply main.js]: Fixed

**Review by Karlfunhouse — APPROVED** [review]
Nice work, let's keep going!

**Pull request merged**